modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -109,6 +110,10 @@ export default {
     serverError: {
       type: Object,
       default: null
+    },
+    docId: {
```

Comment:
Thanks. Is it already passed to object fields?

Reply:
updated to pass it through and confirmed.